doc/nrf/ug_nrf5340.rst

```diff
@@ -496,6 +496,40 @@ Follow these steps to build and program a multi-image build to the nRF5340 appli
 
 #. Program the application sample using :guilabel:`Target` > :guilabel:`Download zephyr/merged.hex`.
 
+Using |VSC| extension
+======================
+
+You can build and program separate images or combined images using |VSC|.
+
+Separate images
+---------------
+
+To build and program only the application core, follow the instructions in `Building an application`_ and use ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuapp_ns`` as the build target.
+
+To build and program a dedicated network sample, follow the instructions in `Building an application`_ and use ``nrf5340dk_nrf5340_cpunet`` as the build target.
```

Comment:
What I understood from here is : http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_nrf5340.html#id18, this would be just for the network core.
I have rephrased it a bit